src/app/modules/users/single-user/single-user.component.ts

```diff
@@ -13,90 +13,217 @@ import { QuestionBase } from '../../admin-shared';
   styleUrls: ['./single-user.component.scss']
 })
 export class SingleUserComponent implements OnInit {
-  questions$: Observable<QuestionBase<any>[]>;
+  questions: any[];
+  questions1 = [
```

Comment:
If you are not using this variable, please delete

Reply:
Removed